src/common.jl

```diff
@@ -10,25 +10,25 @@
 # covariant type notation, i.e. AbstractVector{<:Real}
 #
 
-@compat RealArray{T<:Real,N} = AbstractArray{T,N}
-@compat RealVector{T<:Real} = AbstractArray{T,1}
-@compat RealMatrix{T<:Real} = AbstractArray{T,2}
+const RealArray{T<:Real,N} = AbstractArray{T,N}
```

Comment:
FWIW, `const` isn't needed when there are type parameters -- but using it is fine.